tests/unit/quoted.test.scala

```diff
@@ -5,11 +5,21 @@
 //    ^ constant.numeric.scala
 //      ^ punctuation.section.block.end.scala
 
+   ' 	{ 2 }
```

Comment:
Here I added both a tab and a space to cover the handling of tab characters I have in the code.